tools/check_examples/src/check_examples/check_basic.py

```diff
@@ -23,13 +23,8 @@ def check_markdown(directory: Path, recurse: bool = True) -> int:
     """Check markdown files."""
     return_code = 0
     markdown_files = list_files_of_suffix(directory, ".md", recurse=recurse)
-    check_example_directory = get_git_root_directory() / "tools/check_examples"
-    config_file = check_example_directory / "pymarkdownlnt.toml"
     for file in markdown_files:
         filename = str(file)
-        return_code |= execute(
-            ["pdm", "run", "pymarkdownlnt", "--config", str(config_file.name), "scan", filename],
-            check_example_directory,
-        )
+        return_code |= execute(["markdownlint-cli2", filename], get_git_root_directory())
```

Comment:
```suggestion
        return_code |= execute(["npx markdownlint-cli2", filename], get_git_root_directory())
```

or

```suggestion
        return_code |= execute(["npm run lint:md", filename], get_git_root_directory())
```
if you e.g. create "lint:md" npm script. Or "npm test" if you just call your script test. Anything that makes sense to you works.

Reply:
Done